### PR TITLE
Updated the framework list - phalcon and windwalker

### PR DESCRIPTION
**Pull request by rakesh-sankar**
Added the windwalker and the phalcon to the list. Please review and merge.

### CODE REVIEW

**Comment by pmjones** [comment]
Need a link to the "introduction" of these.

**Comment by rakesh-sankar** [comment]
Here you go:
- For Windwalker - http://windwalker.io/about/
- For Phalcon - http://docs.phalconphp.com/en/latest/index.html

Hope, this helps.

**Comment by pmjones** [comment]
I'm not being clear. I mean links to the first announcements of these projects, in particular Phalcon. Cf. the other links; they don't go to "project pages" they go to "here's the first release and its background" kind of pages.

**Comment by rakesh-sankar** [comment]
Got it, found this for phalcon (http://blog.phalconphp.com/post/100083107700/phalcon-2-beta-3-released & http://blog.phalconphp.com/search/release in general) - hope this helps.

For Windwalker - I couldn't find one.

**Comment by harikt** [comment]
@rakesh-sankar you need to update the links in the commit. 

> For Windwalker - I couldn't find one.

You can remove windwalker then. Pretty new and have not heard.

**Comment by rakesh-sankar** [comment]
Done, updated the Phalcon with the release link and I have removed the windwalker.

@pmjones and @harikt - let me know if that is correct.

File: README.md [hunk 1]
```diff
@@ -62,6 +62,8 @@ Need:
 
 - Laravel 3: <http://maxoffsky.com/code-blog/history-of-laravel-php-framework-eloquence-emerging/>
 
+- Phalcon: <http://blog.phalconphp.com/search/release>
```
Comment (harikt) [inline README.md:65]: You should say Phalcon 1: http://blog.phalconphp.com/post/45916328008/phalcon-1-0-released , so people can see the exact date of release. 1.x series are all same version. 2.x goes to next year. Please see other frameworks like Laravel, Cake, Solar / Aura how they are mentioned.

**Pull request merged**